test/Tests.Infrastructure/RequiresSnowflakeFactAttribute.cs

```diff
@@ -19,4 +19,29 @@ public RequiresSnowflakeFactAttribute()
         if (SnowflakeConnectionString.Instance.CanConnect == false)
             Skip = "Test requires Snowflake database";
     }
+    
+    internal static bool ShouldSkip(out string skipMessage)
```

Comment:
Since we no longer use `RequiresSnowflakeFactAttribute` then maybe let's remove this class completely. I see that ShouldSkip() method for  Azure Queue Storage ETL tests is in a static `Tests.Infrastructure.AzureQueueStorageHelper` class. Maybe we could do the same?

Reply:
done